spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-seata/src/main/java/com/alibaba/cloud/seata/rest/SeataRestTemplateAutoConfiguration.java

```diff
@@ -58,4 +53,14 @@ public void init() {
 		}
 	}
 
+	@Configuration(proxyBeanMethods = false)
```

Comment:
Why did you modify the codes here?

Reply:
Consider guaranteeing the priority of `SeataRestTemplateInterceptor` initialization .